extensions/amp-consent/0.1/amp-consent.js

```diff
@@ -724,11 +724,8 @@ export class AmpConsent extends AMP.BaseElement {
     if (!opt_metadata) {
       return;
     }
-    if (!isObject(opt_metadata) || !opt_consentString) {
-      user().error(
-        TAG,
-        'CMP metadata is invalid or no consent string is found.'
-      );
+    if (!isObject(opt_metadata)) {
```

Comment:
>We always use metadata to transmit the new gdprApplies flag whether there's an existing consent string or not so yes this case is prevalent for us.

If I understand it correctly, the existing string will be erased if it's not provided. Please confirm. 
And if that's the case, I'm not sure that's the desired behavior.

Reply:
Hmm.. good point. This may need to be thought out a little more.

Reply:
Discussed more with publishers, it seems like there is a valid case for this.

https://github.com/ampproject/amphtml/issues/28634#issuecomment-705433038

Reply:
Thanks for confirming. 
You no longer need the `opt_consentString` arg, also please update the method name and comment.